k8s/k8s-crd/src/main.rs

```diff
@@ -0,0 +1,82 @@
+#![deny(rust_2018_idioms)]
+#![warn(clippy::all, clippy::pedantic)]
+#![allow(
+    clippy::let_and_return,
+    clippy::let_unit_value,
+    clippy::missing_errors_doc,
+    clippy::too_many_arguments
+)]
+
+use k8s_openapi::api::core::v1::{Event, EventSource, ObjectReference};
+use k8s_openapi::chrono::Utc;
+use k8s_openapi::apimachinery::pkg::apis::meta::v1::MicroTime;
+use kube::api::PostParams;
+use kube::core::ObjectMeta;
+use uuid::Uuid;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let client = kube::Client::try_default().await?;
+    let events_client: kube::Api<Event> = kube::Api::default_namespaced(client);
+
+    let uuid = Uuid::new_v4();
+    let namespace = "default";
+    let resource_version = "1.0.0";
+    let pod_name = "drw-pod";
+
+    let my_event = Event {
+        type_: Some(String::from("Normal")),
+        action: Some(String::from("Ran sample")),
```

Comment:
This did not show up in the default columns included in `kubectl events`

Reply:
Shows with `kubectl events -o=json`. Not sure what type of value should show up here.